eng/scripts/DownloadDevopsLogger.ps1

```diff
@@ -0,0 +1,6 @@
+New-Item -ItemType directory artifacts -Force
```

Comment:
Should this be in `eng/commom/script` so it can be shared by other languages?

Reply:
It's MSBuild specific logger.